tools/update.hpp

```diff
@@ -110,23 +110,36 @@ namespace antler {
       update_project(CLI::App& app) {
          path = system::fs::current_path().string();
          subcommand = app.add_subcommand("update", "Update an app, dependency, library or test to your project.");
-         subcommand->add_option("-p, path", path, "This must be the path to the `project.yml` or the path containing it.")->default_val(".");
+         subcommand->add_option("-p, path", path, "Path containing the project's yaml file.");
 
-         app_subcommand = subcommand->add_subcommand("app", "Remove app from the project.");
+         app_subcommand = subcommand->add_subcommand("app", "Update an app in the project.");
+         app_subcommand->footer(std::string(R"(Examples:)")
+               + "\n\t" + app.get_name() +R"( update app MyApp C++ \\-O2)"
+               + "\n\t" + app.get_name() +R"( ./path-to-project/ update app -n MyApp --comp -O2)");

```

Comment:
```suggestion
               + "\n\t" + app.get_name() +R"( update app -p ./path-to-project/ -n MyApp --comp -O2)");

```

Reply:
However when I'm running above command I get: https://github.com/AntelopeIO/antler-proj/issues/38

Reply:
#38 corrected in c4c355b5c17acaa5e054b2f51844f4200c0a517a

Reply:
Regarding location of the path, both lines are valid. I'm inclined to just leave it as is.

Reply:
Original version doesn't work for me (while new works):
```
mikel@msi:~/repo/antler-proj/build/tools$ ./antler-proj  ./ update app  -n myapp2 --comp -O3
The following argument was not expected: ./
Run with --help for more information.
```